internal/fusefrontend_reverse/rfs.go

```diff
@@ -417,10 +380,14 @@ func (rfs *ReverseFS) StatFs(relPath string) *fuse.StatfsOut {
 
 // Readlink - FUSE call
 func (rfs *ReverseFS) Readlink(relPath string, context *fuse.Context) (string, fuse.Status) {
-	if rfs.isExcluded(relPath) {
+	excluded, pPath, err := rfs.isExcludedCipher(relPath)
```

Comment:
The behavior here is also changed for special files (translated config, dirIV and name files).

Reply:
This is pretty much the same issue as above. Previously `decryptPath` (in `openBackingDir`) would have failed for files ending with `.name` or other special files (e.g., `gocryptfs.diriv`). Such files should never be symlinks.

With the new code, however, the `.name` extension is silently dropped. If the user would somehow trick FUSE into issuing a `Readlink` operation on a name file, it would now return the result as if the command would have been applied on the file without the `.name` extension.